src/main/java/com/google/sps/servlets/getData.java

```diff
@@ -2,28 +2,19 @@
    getData.java
 
```

Comment:
Hi! Here I change a bit of the description just to keep it up to date with what it does. (No longer uses hashmap, etc)